design/cli-install-changes.md

```diff
@@ -116,7 +116,7 @@ Commands/flags for backup locations.
       set
         --default string                                  sets the default backup storage location (default "default") (NEW, -- was `server --default-backup-storage-location; could be set as an annotation on the BSL)
         --credentials mapStringString                     sets the name of the corresponding credentials secret for a provider. Format is provider:credentials-secret-name. (NEW)
-        --cacert-file mapStringString                     configuration to use for creating a secret containing a custom certificate for an S3 location of a plugin provider. Format is provider:path-to-file. (NEW)
+        --cacert mapStringString                          configuration to use for creating a secret containing a custom certificate for an S3 location of a plugin provider. Format is provider:path-to-file. (NEW)
```

Comment:
You can remove this edit - the design docs don't have to stay exactly in sync with their implementation.